specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsCreateOrUpdate.json

```diff
@@ -13,19 +13,21 @@
     "200": {
       "body": {
         "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/rg1/providers/Microsoft.Aadiam/azureADMetrics/ddb1",
-        "name": "azureADMetricsName",
-        "type": "Microsoft.Aadiam/azureADMetrics",
+        "location": "West US",
+        "name": "ddb1",
+        "provisioningState": "Succeeded",
         "tags": {},
-        "location": "West US"
+        "type": "Microsoft.Aadiam/azureADMetrics"
       }
     },
     "201": {
       "body": {
         "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/rg1/providers/Microsoft.Aadiam/azureADMetrics/ddb1",
-        "name": "azureADMetricsName",
-        "type": "Microsoft.Aadiam/azureADMetrics",
+        "location": "West US",
+        "name": "ddb1",
+        "provisioningState": "Created",
```

Comment:
This needs to be one level deeper.  It should be "properties": { "provisioningState": "value" }

Reply:
When I nested it under properties, I got a Model Validation Error saying "additional property "properties" is not allowed".

Reply:
'0': {
    code: 'RESPONSE_VALIDATION_ERROR',
    id: 'OAV108',
    message: 'Found errors in validating the response with statusCode "200" for x-ms-example "azureADMetricsUpdate" in operation "azureADMetrics_Update".',
    innerErrors: [
      {
        code: 'INVALID_RESPONSE_BODY',
        errors: [
          {
            code: 'OBJECT_ADDITIONAL_PROPERTIES',
            params: [ 'properties' ],
            message: 'Additional properties not allowed: properties',
            path: [],
            title: '#/definitions/azureADMetricsConfig',
            description: 'AzureADMetrics resource.',
            position: {
              line: 325,
              column: 29,
              directives: { RequiredPropertiesMissingInResourceModel: '.*' }
            },
            url: 'C:\\Project\\Forked\\azure-rest-api-specs\\specification\\azureactivedirectory\\resource-manager\\Microsoft.Aadiam\\preview\\2020-07-01-preview\\azureADMetrics.json',
            directives: {
              R3020: '.*',
              R3023: '.*',
              RequiredPropertiesMissingInResourceModel: '.*'
            },
            jsonPosition: { line: 14, column: 15 },
            jsonUrl: 'C:/Project/Forked/azure-rest-api-specs/specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsUpdate.json'
          }
        ],
        message: 'Invalid body: Additional properties not allowed: properties',
        path: []
      }
    ]
  },

Reply:
You have to change the definition so that it is nested under properties.  That error is just saying the example can't add random properties that aren't in the definition.

Reply:
The definition is nested under properties within azureADConfig.
```
    "azureADMetricsConfig": {
      "description": "AzureADMetrics resource.",
      "type": "object",
      "properties": {
        "provisioningState": {
          "description": "The provisioning state of the resource.",
          "type": "string"
        }
```